BlueprintUI/Sources/Element/Accessibility.swift

```diff
@@ -0,0 +1,202 @@
+import UIKit
+
+public struct Accessibility {} //  Namespace for collecting Blueprint models of UIKit Accessibility features.
+
+// MARK: - Accessibility Traits
+extension Accessibility {
+    public enum Trait {
+        /// Used in conjunction with UIAccessibilityTrait.adjustable, these will be called to allow accessible adjustment of a value, for example in a slider or stepper control.
+        /// See [Accessibility Increment Documentation](https://developer.apple.com/documentation/objectivec/nsobject/1615076-accessibilityincrement) for further information.
+        public typealias IncrementAction = () -> Void
+
+        /// Used in conjunction with UIAccessibilityTrait.adjustable, these will be called to allow accessible adjustment of a value, for example in a slider or stepper control.
+        /// See [Accessibility Decrement Documentation](https://developer.apple.com/documentation/objectivec/nsobject/1615169-accessibilitydecrement) for further information.
+        public typealias DecrementAction = () -> Void
+
+        case button
+        case link
+        case header
+        case searchField
+        case image
+        case selected
+        case playsSound
+        case keyboardKey
+        case staticText
+        case summaryElement
+        case notEnabled
+        case updatesFrequently
+        case startsMediaSession
+        case adjustable(IncrementAction, DecrementAction)
+        case allowsDirectInteraction
+        case causesPageTurn
+        case tabBar
+        case backButton
```

Comment:
Mind copying over the test from the Market PR so if this value changes the test fails? https://github.com/squareup/market/pull/8334/files#diff-4365aee19fb253d8a127c10e4b0ab7cb346642061b5070ff2a01e21c2bbd9231

Reply:
I tried that initially but couldn't get it working in blueprint. i'll bring it up on slack.